SPECS/ceph/CVE-2020-14378.patch

```diff
@@ -0,0 +1,55 @@
+From a70de0405b30f576c7c947098775b07bcdcbbce2 Mon Sep 17 00:00:00 2001
+From: archana25-ms <v-shettigara@microsoft.com>
+Date: Wed, 26 Mar 2025 07:08:51 +0000
+Subject: [PATCH] Address CVE-2020-14378
+
+Upstream Patch Reference: https://git.dpdk.org/dpdk-stable/commit/?id=7a5af91f8bf46f121cc1a7873045ef37f63d56c2
```

Comment:
Patch applies with a shift.
Same patch applicable to two files of same project in two different directories.